src/Microsoft.DotNet.Darc/tests/Microsoft.DotNet.Darc.Tests/DependencyAddUpdateTests.cs

```diff
@@ -1,5 +1,8 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
```

Comment:
Why... this file and not the others?  Should we add this to "DependencyTestDriver" too?

Reply:
Probably because I was doing this opportunistically and didn't notice the others.